src/80211_if.c

```diff
@@ -2941,6 +2974,13 @@ static int uccp420wlan_init(void)
 		}
 	}
 
+	error=start_core(hw);
```

Comment:
Apart from firmware loading the bt_state needs to be set (**CALL_UMAC(uccp420wlan_prog_btinfo, dev->params->bt_state);)**to make the BT scan work,  for that we need to initialize the LMAC hence I segregated the code that just loads the fw and initializes the lmac and sets the BT state in start_core the rest of the code can be left in the start() functions.